smithy-swift-codegen/src/test/kotlin/mocks/MockHttpRestXMLProtocolGenerator.kt

```diff
@@ -65,7 +65,7 @@ class MockHttpRestXMLProtocolGenerator : HttpBindingProtocolGenerator() {
         writer: SwiftWriter,
         defaultTimestampFormat: TimestampFormatTrait.Format,
     ) {
-        val decodeGenerator = StructDecodeXMLGenerator(ctx, members, mapOf(), writer, defaultTimestampFormat)
+        val decodeGenerator = StructDecodeXMLGenerator(ctx, members, shapeMetadata, writer, defaultTimestampFormat)
         decodeGenerator.render()
     }
     override fun generateProtocolUnitTests(ctx: ProtocolGenerator.GenerationContext): Int {
```

Comment:
Next pr will consist of moving removing this class, and moving all of our restxml unit tests over to aws-sdk-swift.